src/views/AddLiquidity.vue

```diff
@@ -269,6 +270,15 @@ export default class AddLiquidity extends Mixins(LoadingMixin, TokenPairMixin) {
     return prevPosition
   }
 
+  get strategicBonusApy (): Nullable<string> {
```

Comment:
It seems that this block of code is the same with src/components/ConfirmTokenPairDialog.vue. Could we have it in one place please? Maybe mixing is a good candidate for this.

Reply:
we've decided to keep it as it is